### PR TITLE
HoP and Captain now start wearing locker contents +HUDs for captains.

### PR DESCRIPTION
**Pull request by Sarcalogo**
May as well start the round with it on because all people who pick these
jobs immediately equip the items.

### CODE REVIEW

**Comment by cacogen** [comment]
as a note, this will mean latejoining captains and hops will spawn with their guns and they will no longer be available to be stolen from their lockers by traitors or people with stolen access

**Comment by Sarcalogo** [comment]
Yep pretty much.

**Comment by JohnGinnane** [comment]
Code is good but not sure I agree with the idea, need head input as I know this was discussed quite a bit in oldcode

File: code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -32,15 +32,15 @@ Captain
 
 	id = /obj/item/weapon/card/id/gold
 	belt = /obj/item/device/pda/captain
-	glasses = /obj/item/clothing/glasses/sunglasses
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
```
Comment (JohnGinnane) [inline code/modules/jobs/job_types/captain.dm:35]: Suddenly cap spawns with sec classes? Very shneaky

**Comment by McBawbaggings** [comment]
needs changelog?
Not a big fan of HoP getting his shit right off the bat since Captain can technically appoint a HoP.

**Comment by BTFO** [comment]
**_P O W E R
C R E E P_**

**Comment by KittyKathy** [comment]
This is a bad idea, also no captain should not spawn with sechuds they are not security the detective is more apart of security than the captain but he gets no sechuds. This is lets make the captain HOS 2.0 the pr so it's easier from them to validhunt/powergame without having to spend time getting some of the gear. Pretty much the same for the HOP they can get their gear from their locker if they bother to spend 1 minute in their office to maybe do their job instead of running around as batman.

**Comment by cacogen** [comment]
it'd be nice if locker contents/spawn items could vary based on whether they were there roundstart or not

**Comment by Sarcalogo** [comment]
On what basis do you say that the Captain, the person who rules over the entire station, has the power to verify executions, whose headset has access to the security channel default enabled,  is not part of Security?

**Comment by JohnGinnane** [comment]
Because he hands out those duties to the head of security. It's their job to take care of the tide (from a lore pov)

**Comment by JohnGinnane** [comment]
At this rate we might as well give him hos egun and shield as well...

**Comment by Sarcalogo** [comment]
It is of vital importance that the captain know who is and is not implanted. I don't think I've ever seen a captain not get SECHUD's almost immediately, and he spawned with them in his locker on the previous code. On the previous code, Captain also spawned with no sunglasses on, on here he does, but just regular sunglasses. It's a combination of those two factors. And really, nothing worse than late joining as the captain and getting shot by some guy in full HoS armor who just so happened to be an unimplanted guy in HoS stuff.

**Comment by McBawbaggings** [comment]
I don't really mind condoms getting sec huds off of the bat. Giving them their tools and making their lockers not worth the while is a different issue.

**Comment by JohnGinnane** [comment]
Cap had sechud in his locker?

**Comment by Spacedong** [comment]
yeah but hop didn't

File: code/modules/jobs/job_types/captain.dm (hunk 1 above)
Comment (Spacedong) [inline code/modules/jobs/job_types/captain.dm]: >trying to sneak code in

**Comment by Spacedong** [comment]
the day we accept sarcalago PRs the day we kill the server

**Comment by JohnGinnane** [comment]
Right so cap having sechud would be oldcode again which is fine, as for starting with all this equipment, still not sure about it

**Comment by Sarcalogo** [comment]
The only thing else I'm doing is giving the Captain his energy gun at round start so he doesn't join, get shot, and immediately die, same with HoP.

If you'd like, I could try to figure out how to spawn two lockers. One locker will be spawned if the station has no Captain, it will start with all of his default equipment. The other locker will be spawned if there is a Captain, and it will not have the equipment that the Captain now has on him (E-gun and glasses). Same for the HoP. Is that better?

**Comment by JohnGinnane** [comment]
But then if there's no cap then the sweet gear is missing from his office... What's the objective of giving them their gear? What's wrong with how it was?

**Comment by Sarcalogo** [comment]
No, you're not understand what I said correctly. 

No Captain = His locker exists with everything in it.
Captain = He starts with his stuff equipped and the locker that spawns is a different locker, one with less stuff.

Then you and I can both have our way.

**Comment by cacogen** [comment]
do the locker thing, best of both worlds

**Comment by Sarcalogo** [comment]
Okay then. Closing this... will do a seperate pull request with Captain spawning with SEC HUD's cause that's gotta get in immediately. Rest will take time for me to figure out how to do anyway.

**Comment by JohnGinnane** [comment]
Whoops, yeah I misunderstood

**Pull request closed without merging**